src/KubeOps/Operator/Services/IResourceTypeService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace KubeOps.Operator.Services
+{
+    public interface IResourceTypeService
+    {
+        public void AddAssembly(Assembly assembly);
+
+        public IEnumerable<Type> GetResourceTypesByAttribute<TAttribute>()
```

Comment:
```suggestion
        IEnumerable<Type> GetResourceTypesByAttribute<TAttribute>()
```

default interfaces are all public